site/docs/components/button/examples.mdx

```diff
@@ -10,36 +10,48 @@ data:
 <LivePreviewControls>
 <LivePreview componentName="button" exampleName="CTA" >
 
+### CTA
+
 Use the CTA button for high priority actions.
 
 </LivePreview>
 
 <LivePreview componentName="button" exampleName="Primary" >
 
+### Primary
+
 Use the primary button for routine, non-urgent actions.
 
 </LivePreview>
 
 <LivePreview componentName="button" exampleName="Secondary" >
 
+### Secondary
+
 Use the secondary button for non-critical actions that support the user but do not impact a flow.
 
 </LivePreview>
 
-<LivePreview componentName="button" exampleName="IconAndLabel" >
+<LivePreview componentName="button" exampleName="IconAndLabel" title="Icon and label">
```

Comment:
Because the auto-generated one has the wrong capitalisation. It would become "Icon And Label" rather than "Icon and label" and our content style guide requires headings to use sentence case.